src/features/ITGovernance/Requester/SystemIntake/Confirmation/index.tsx

```diff
@@ -1,34 +1,105 @@
 import React from 'react';
 import { useTranslation } from 'react-i18next';
-import { Link, useParams } from 'react-router-dom';
-import { Icon } from '@trussworks/react-uswds';
+import { Link as ReactRouterLink, useParams } from 'react-router-dom';
+import {
+  Breadcrumb,
+  BreadcrumbBar,
+  BreadcrumbLink,
+  ButtonGroup,
+  GridContainer,
+  SummaryBox,
+  SummaryBoxContent,
+  SummaryBoxHeading
+} from '@trussworks/react-uswds';
 
+import ExternalLinkAndModal from 'components/ExternalLinkAndModal';
 import PageHeading from 'components/PageHeading';
 
-const Confirmation = () => {
+const Confirmation = ({
+  submissionSuccess
+}: {
+  submissionSuccess: boolean;
+}) => {
   const { systemId } = useParams<{ systemId: string }>();
   const { t } = useTranslation('intake');
 
   return (
-    <div className="grid-container margin-bottom-7">
-      <div>
-        <PageHeading>{t('submission.confirmation.heading')}</PageHeading>
-        <h2 className="margin-bottom-8 text-normal">
-          {t('submission.confirmation.subheading', {
-            referenceId: systemId
-          })}
-        </h2>
-        <div>
-          <Link
-            to={`/governance-task-list/${systemId}`}
-            className="display-flex"
+    <>
+      <div
+        className={`margin-bottom-8  padding-bottom-6 ${submissionSuccess ? 'bg-success-lighter' : 'bg-error-lighter'}`}
+      >
+        <GridContainer>
+          <BreadcrumbBar
+            variant="wrap"
+            className={`margin-bottom-6 ${submissionSuccess ? 'bg-success-lighter' : 'bg-error-lighter'}`}
           >
-            <Icon.NavigateBefore className="margin-x-05" aria-hidden />
-            {t('submission.confirmation.taskListCta')}
-          </Link>
-        </div>
+            <Breadcrumb>
+              <BreadcrumbLink asCustom={ReactRouterLink} to="/">
+                <span>{t('taskList:navigation.home')}</span>
+              </BreadcrumbLink>
+            </Breadcrumb>
+            <Breadcrumb>
+              <BreadcrumbLink
+                asCustom={ReactRouterLink}
+                to={`/governance-task-list/${systemId}`}
+              >
+                <span>{t('taskList:navigation.governanceTaskList')}</span>
+              </BreadcrumbLink>
+            </Breadcrumb>
+            <Breadcrumb current>
+              {t('taskList:navigation.intakeRequest')}
+            </Breadcrumb>
+          </BreadcrumbBar>
+
+          <PageHeading className="margin-top-0 margin-bottom-1">
+            {submissionSuccess
+              ? t('submission.success.heading')
+              : t('submission.error.heading')}
+          </PageHeading>
+          <p className="font-body-lg line-height-body-5 margin-top-0 margin-bottom-5">
```

Comment:
This text needs the `text-light` util class to match the Figma.
```suggestion
          <p className="font-body-lg line-height-body-5 margin-top-0 margin-bottom-5 text-light">
```